Sources/SmileID/Classes/BiometricKYC/OrchestratedBiometricKycViewModel.swift

```diff
@@ -193,6 +194,8 @@ class OrchestratedBiometricKycViewModel: ObservableObject {
             else {
                 throw error
             }
+            networkRetries += 1
```

Comment:
I think we also need increase the network retries here https://github.com/smileidentity/ios/blob/0a8712441702a1845956a8136ce73362d12bc856/Sources/SmileID/Classes/BiometricKYC/OrchestratedBiometricKycViewModel.swift#L58 - for the manual retries. At the moment, it only counts in case there is a network error for the prepUpload.

Reply:
ie. before the submitJob